test/gas-analytics/HubPool.RefundExecution.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  toBNWei,
+  toBN,
+  SignerWithAddress,
+  seedWallet,
+  Contract,
+  ethers,
+  getContractFactory,
+  BigNumber,
+  randomAddress,
+  expect,
+} from "../utils";
```

Comment:
I think it's cleaner to do two imports for this kind of thing, rather than taking 12 lines.

Reply:
If you do `import * as utils from "../utils"` then you need to call `utils.expect` and `utils.Contract` for types, which seems annoying. Were you thinking of something different?